NineChronicles.Headless/Properties/NineChroniclesNodeServiceProperties.cs

```diff
@@ -42,7 +42,65 @@ public class NineChroniclesNodeServiceProperties
 
         public int TxQuotaPerSigner { get; set; }
 
-
+        public static LibplanetNodeServiceProperties<NineChroniclesActionType>
+            GenerateLibplanetNodeServiceProperties(
+                string? appProtocolVersionToken = null,
+                string? genesisBlockPath = null,
+                string? swarmHost = null,
+                ushort? swarmPort = null,
+                string? swarmPrivateKeyString = null,
+                string? storeType = null,
+                string? storePath = null,
+                int storeStateCacheSize = 100,
+                string[]? iceServerStrings = null,
+                string[]? peerStrings = null,
+                string[]? trustedAppProtocolVersionSigners = null,
+                bool noMiner = false,
+                bool render = false,
+                int workers = 5,
+                int confirmations = 0,
+                bool nonblockRenderer = false,
+                int nonblockRendererQueue = 512,
+                int messageTimeout = 60,
+                int tipTimeout = 60,
+                int demandBuffer = 1150,
+                string[]? staticPeerStrings = null,
+                bool preload = true,
+                int minimumBroadcastTarget = 10,
+                int bucketSize = 16,
+                string chainTipStaleBehaviorType = "reboot",
+                int maximumPollPeers = int.MaxValue)
+        {
```

Comment:
This was just to make it compatible with the current version so that it could be used without replacing anything except the NineChronicles.Headless.dll.